src/mongocrypt-key.c

```diff
@@ -23,7 +23,9 @@ static bool
 _one_key_alt_name_equal (_mongocrypt_key_alt_name_t *ptr_a,
                          _mongocrypt_key_alt_name_t *ptr_b)
 {
+   BSON_ASSERT_PARAM (ptr_a);
    BSON_ASSERT (ptr_a->value.value_type == BSON_TYPE_UTF8);
+   BSON_ASSERT_PARAM (ptr_b);
```

Comment:
```suggestion
   BSON_ASSERT_PARAM (ptr_a);
   BSON_ASSERT_PARAM (ptr_b);
   BSON_ASSERT (ptr_a->value.value_type == BSON_TYPE_UTF8);
```
Suggest grouping preconditions at top of function before code.